docs/graphql-api.md

```diff
@@ -183,6 +201,21 @@ The GraphQL spec defines `Int` and `Float` to have sizes of 32 bytes.
 
 This API additionally includes a `BigInt` number type to represent arbitrarily large integer numbers.
 
+## 3.4 Enums 
+
+You can also create `enums` within a schema. Enums have the following syntax:
+
+```graphql
+enum TokenStatus {
+    OriginalOwner,
+    SecondOwner,
+    ThirdOwner,
+}
+```
+
+In order to set a store value with an enum, you simply pass the string value of the result. In the example above, you can set the `TokenStatus` to the second owner with `"SecondOwner"`. 
```

Comment:
`pass the string value of the result` -> `use the name of the enum value as a string`.